CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Regex terminal hints ([see features.md](./docs/features.md#hints))
 - macOS keybinding (cmd+alt+H) hiding all windows other than Alacritty
 - Support for `magnet` URLs
+- Support for `ipfs`/`ipns` URLs
```

Comment:
Could you add a new `0.9.0-dev` section at the top and add things there? And then remove the `-dev` from `0.8.0-dev`.